### ISSUE TITLE
[PATCH] Fix typos (spelling errors) in cpan/CGI/*.

### ISSUE THREAD

**Issue by p5pRT** [Perl/perl5#10952]
Migrated from [rt.perl.org#81780](https://rt-archive.perl.org/perl5/Ticket/Display.html?id=81780) (status was 'rejected')

Searchable as RT81780$

**Comment by p5pRT** [comment]
### From @pjacklam



---
 cpan/CGI/Changes           |   16 \+\+\+\+\+\+\+\+\-\-\-\-\-\-\-\-
 cpan/CGI/lib/CGI\.pm        |    9 \+\+\+\+\+\-\-\-\-
 cpan/CGI/lib/CGI/Pretty\.pm |    2 \+\-
 cpan/CGI/lib/CGI/Util\.pm   |    2 \+\-
 cpan/CGI/t/cookie\.t        |   14 \+\+\+\+\+\+\+\-\-\-\-\-\-\-
 5 files changed\, 22 insertions\(\+\)\, 21 deletions\(\-\)

<details><summary>Inline Patch</summary>

```diff
diff --git a/cpan/CGI/Changes b/cpan/CGI/Changes
index 0647ef5..6b71ed2 100644
--- a/cpan/CGI/Changes
+++ b/cpan/CGI/Changes
@@ -145,7 +145,7 @@ Version 3.46
   13.We now give objects passed to CGI::Carp::die a chance to be stringified.
      Thanks to teek and Yanick Champoux (RT#41530)
   14.Turning off autoEscape() now only affects the behavior of built-in HTML
-     generation fuctions. Explicit calls to escapeHTML() always escape HTML regardless
+     generation functions. Explicit calls to escapeHTML() always escape HTML regardless
      of the setting. Thanks to vindex, Bob Kuo and Mark Stosberg (RT#40748)
   15.In CGI::Fast, preferences set via pragmas are now preserved.
      Thanks to heinst and Mark Stosberg (RT#32119)
@@ -314,7 +314,7 @@ Version 3.45
   Version 3.25
   1. Fixed the link to the Netscape frames page.
   2. Added ability to specify an alternate stylesheet.
-  3. Add support for XForms POST submssion both as application/xml or as multipart/related
+  3. Add support for XForms POST submission both as application/xml or as multipart/related
 
   Version 3.24
   1. In startform(), if request_uri() returns undef, then falls back
@@ -585,7 +585,7 @@ Version 3.45
 
   Version 2.93
 
-    1. Fixed embarassing bug in mp1 support.
+    1. Fixed embarrassing bug in mp1 support.
 
   Version 2.92
 
@@ -1137,7 +1137,7 @@ Version 3.45
    21. Fixed warning in initialize_globals() under mod_perl.
    22. File uploads from Macintosh versions of MSIE should now work.
    23. Pragmas now preceded by dashes (-nph) rather than colons (:nph).
-       Old style is supported for backward compatability.
+       Old style is supported for backward compatibility.
    24. Can now pass arguments to all functions using {} brackets,
        resolving historical inconsistencies.
    25. Removed autoloader warnings about absent MultipartBuffer::DESTROY.
@@ -1205,7 +1205,7 @@ Version 3.45
 
     1. Fixed a warning about an undefined environment variable.
     2. Doug's patch for redirect() under mod_perl
-    3. Partial fix for busted inheritence from CGI::Apache
+    3. Partial fix for busted inheritance from CGI::Apache
     4. Documentation fixes.
 
   Version 2.32
@@ -1274,7 +1274,7 @@ Version 3.45
     1. Changed behavior of scrolling_list(), checkbox() and
        checkbox_group() methods so that defaults are honored correctly.
        The "fix" causes endform() to generate additional <INPUT
-       TYPE="HIDDEN"> tags -- don't be surpised.
+       TYPE="HIDDEN"> tags -- don't be surprised.
     2. Fixed bug involving the detection of the SSL protocol.
     3. Fixed documentation error in position of the -meta argument in
        start_html().
@@ -1369,7 +1369,7 @@ Version 3.45
 
     1. Added cookie() support routines.
     2. Added -expires parameter to header().
-    3. Added cgi-lib.pl compatability mode.
+    3. Added cgi-lib.pl compatibility mode.
     4. Made the module more configurable for different operating systems.
     5. Fixed a dumb bug in JavaScript button() method.
 
@@ -1499,7 +1499,7 @@ Version 3.45
 
     1. The user_agent() method is now documented;
     2. A potential security hole in import() is now plugged.
-    3. Changed name of import() to import_names() for compatability with
+    3. Changed name of import() to import_names() for compatibility with
        CGI:: modules.
 
   Bug fixes in version 1.53
diff --git a/cpan/CGI/lib/CGI.pm b/cpan/CGI/lib/CGI.pm
index d320d7f..a77e108 100644
--- a/cpan/CGI/lib/CGI.pm
+++ b/cpan/CGI/lib/CGI.pm
@@ -2062,7 +2062,7 @@ END_OF_FUNC
 #   $name -> Name of the text field
 #   $default -> Optional default value of the field if not
 #                already defined.
-#   $size ->  Optional width of field in characaters.
+#   $size ->  Optional width of field in characters.
 #   $maxlength -> Optional maximum number of characters.
 # Returns:
 #   A string containing a <input type="text"> field
@@ -2078,7 +2078,7 @@ END_OF_FUNC
 #### Method: filefield
 # Parameters:
 #   $name -> Name of the file upload field
-#   $size ->  Optional width of field in characaters.
+#   $size ->  Optional width of field in characters.
 #   $maxlength -> Optional maximum number of characters.
 # Returns:
 #   A string containing a <input type="file"> field
@@ -3003,9 +3003,9 @@ END_OF_FUNC
 # PATH_INFO=/http://foo when the user requests
 # http://xxx/script.cgi/http://foo
 #
-# Old versions of this module used to accomodate with those scripts, so
+# Old versions of this module used to accommodate with those scripts, so
 # this is why we do this here to keep those scripts backward compatible.
-# Basically, we accomodate with those scripts but within limits, that is
+# Basically, we accommodate with those scripts but within limits, that is
 # we only try to preserve the number of / that were provided by the user
 # if $REQUEST_URI and "$SCRIPT_NAME$PATH_INFO" only differ by the number
 # of consecutive /.
@@ -7613,6 +7613,7 @@ Returns the remote host IP address, or
 127.0.0.1 if the address is unavailable.
 
 =item B<script_name()>
+
 Return the script name as a partial URL, for self-referring
 scripts.
 
diff --git a/cpan/CGI/lib/CGI/Pretty.pm b/cpan/CGI/lib/CGI/Pretty.pm
index 869fe0c..c4ee85f 100644
--- a/cpan/CGI/lib/CGI/Pretty.pm
+++ b/cpan/CGI/lib/CGI/Pretty.pm
@@ -168,7 +168,7 @@ sub initialize_globals {
     # This is the string used for indentation of tags
     $CGI::Pretty::INDENT = "\t";
     
-    # This is the string used for seperation between tags
+    # This is the string used for separation between tags
     $CGI::Pretty::LINEBREAK = $/;
 
     # These tags are not prettify'd.
diff --git a/cpan/CGI/lib/CGI/Util.pm b/cpan/CGI/lib/CGI/Util.pm
index ef95c9f..f856594 100644
--- a/cpan/CGI/lib/CGI/Util.pm
+++ b/cpan/CGI/lib/CGI/Util.pm
@@ -240,7 +240,7 @@ sub unescape {
 # Byte strings were traditionally used directly as a sequence of octets.
 # This worked if they actually represented binary data (i.e. in CGI::Compress).
 # This also worked if these byte strings were actually utf-8 encoded; e.g.,
-# when the source file used utf-8 without the apropriate "use utf8;".
+# when the source file used utf-8 without the appropriate "use utf8;".
 # This fails if the byte string is actually a Latin 1 encoded string, but it
 # was always so and cannot be fixed without breaking the binary data case.
 # -- Stepan Kasal <skasal@redhat.com>
diff --git a/cpan/CGI/t/cookie.t b/cpan/CGI/t/cookie.t
index 8249f07..78a8a75 100644
--- a/cpan/CGI/t/cookie.t
+++ b/cpan/CGI/t/cookie.t
@@ -175,12 +175,12 @@ my @test_cookie = (
   is($c->name   , 'baz', 'name is correct');
   is($c->value  , 'qux', 'value is correct');
   ok(!defined $c->expires,       'expires is not set');
-  ok(!defined $c->domain ,       'domain attributeis not set');
-  is($c->path, '/',      'path atribute is set to default');
+  ok(!defined $c->domain ,       'domain attributes not set');
+  is($c->path, '/',      'path attribute is set to default');
   ok(!defined $c->secure ,       'secure attribute is set');
   ok( !defined $c->httponly, 'httponly attribute is not set' );
 
-# I'm really not happy about the restults of this section.  You pass
+# I'm really not happy about the results of this section.  You pass
 # the new method invalid arguments and it just merilly creates a
 # broken object :-)
 # I've commented them out because they currently pass but I don't
@@ -193,8 +193,8 @@ my @test_cookie = (
 #    is($c->name   , 'baz',     'name is correct');
 #    ok(!defined $c->value, "Value is undefined ");
 #    ok(!defined $c->expires, 'expires is not set');
-#    ok(!defined $c->domain , 'domain attributeis not set');
-#    is($c->path   , '/', 'path atribute is set to default');
+#    ok(!defined $c->domain , 'domain attributes not set');
+#    is($c->path   , '/', 'path attribute is set to default');
 #    ok(!defined $c->secure , 'secure attribute is set');
 
 }
@@ -322,8 +322,8 @@ my @test_cookie = (
   is($c->name,          'Clash', 'name now returns updated value');
 
   # this is insane!  it returns a simple scalar but can't accept one as
-  # an argument, you have to give it an arrary ref.  It's totally
-  # inconsitent with these other methods :-(
+  # an argument, you have to give it an array ref.  It's totally
+  # inconsistent with these other methods :-(
   is($c->value,           'Hamster', 'value is correct');
   is($c->value(['Gerbil']), 'Gerbil',  'value is set correctly');
   is($c->value,           'Gerbil',  'value now returns updated value');
-- 
1.7.3.3
```
</details>

**Comment by p5pRT** [comment]
@iabyn - Status changed from 'new' to 'rejected'
